src/main/java/controller/RacingGameController.java

```diff
@@ -0,0 +1,75 @@
+package controller;
+
+import domain.RacingCar;
+import domain.RacingCars;
+import domain.TryCount;
+import exception.CommaNotFoundException;
+import exception.NameIsOutOfBoundException;
+import exception.NotPositiveIntegerException;
+import java.util.List;
+import java.util.stream.Collectors;
+import util.CommaSeparator;
+import view.InputView;
+import view.OutputView;
+
+public class RacingGameController {
+
+    private RacingCars racingCars;
+    private TryCount tryCount;
+
+    public void start() {
+        setUpGame();
+        playGame();
+    }
+
+    private void setUpGame() {
+        racingCars = createRacingCar();
+        tryCount = getTryCount();
+    }
+
+    private RacingCars createRacingCar() {
+        List<String> names;
+        try {
```

Comment:
```suggestion
    private RacingCars createRacingCar() {
        try {
                  List<String> names = ...
```



`names` 를 catch 내부에서 사용하려했던걸까요?
지금은 쓰이지 않는 것 같으니 try 내부로 넣어주면 가독성이 더 올라갈 것 같아요 😄

Reply:
미쳐 발견하지 못했었네요 바로 수정했습니다 감사합니다!